### PR TITLE
domain: avoid check schema changed when change is tiflash replica status

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Some DDL change no need to add to `schemaChecker`.
Such as:
* ActionUpdateTiFlashReplicaStatus
* ActionSetTiFlashReplica

Maybe below DDL type also no need to add to `schemaChecker`:
* ActionCreateSchema
* ActionCreateTable
* ActionShardRowID
* ActionModifyTableComment
* ActionCreateView
* ActionRecoverTable
* ActionRepairTable

### Check List 

Tests 

 - 

Code changes

 - Has exported function/method change

Side effects

 - Possible performance regression

Related changes

 - Need to cherry-pick to the release branch

Release note

 - Write release note for bug-fix or new feature.

### CODE REVIEW

**Comment by Deardrops** [comment]
/run-all-tests tidb-test=pr/915
Don't mind, this is for another test.

**Comment by Deardrops** [comment]
/run-all-tests tidb-test=pr/933
Don't mind, this is for another test.

File: domain/domain.go [hunk 1]
```diff
@@ -254,6 +254,9 @@ func (do *Domain) tryLoadSchemaDiffs(m *meta.Meta, usedVersion, newVersion int64
 		if err != nil {
 			return false, nil, err
 		}
+		if diff.Type == model.ActionUpdateTiFlashReplicaStatus || diff.Type == model.ActionSetTiFlashReplica {
```
Comment (bb7133) [inline domain/domain.go]: Since we may add more skippable types, how about adding a function like `skippableSchemaCheckTypes(diff.Type)`?

Comment (zimulala) [inline domain/domain.go]: After calling `tryLoadSchemaDiffs`, we will call `schemaValidator.Update` to affect `latestSchemaVer`.  So this  `latestSchemaVer` will still be added to the deltaSchemaInfos queue. I don't think this change can get the optimization you want.

**Comment by Deardrops** [comment]
/run-all-tests tidb-test=pr/933
Don't mind, this is for another test.

**Comment by Deardrops** [comment]
/run-all-tests tidb-test=pr/933
Don't mind, this is for another test.

File: domain/domain.go [hunk 2]
```diff
@@ -254,12 +254,23 @@ func (do *Domain) tryLoadSchemaDiffs(m *meta.Meta, usedVersion, newVersion int64
...
+		}
 		tblIDs = append(tblIDs, ids...)
 	}
 	builder.Build()
 	return true, tblIDs, nil
 }
 
+func canSkipSchemaCheckerDDL(tp model.ActionType) bool {
+	switch tp {
+	case model.ActionUpdateTiFlashReplicaStatus, model.ActionSetTiFlashReplica:
```
Comment (AilinKid) [inline domain/domain.go]: two options here, how about use if A||B? maybe fast.

Comment (crazycs520) [reply domain/domain.go]: em... I prefer `switch`😂

**Pull request merged**